CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 ## v1.4.0
 
 - [core] fixed handling of environment variables on Windows [#7973](https://github.com/eclipse-theia/theia/pull/7973)
+- [core] added functionality for handling context menu for `tab-bars` without changing focus.[#6965](https://github.com/eclipse-theia/theia/pull/6965)
+
+Breaking Changes:
+
+- [core] Context-menu for `tab-bars` requires an `Event` to be passed onto it to perform actions without changing focus [#6965](https://github.com/eclipse-theia/theia/pull/6965)
+  - Removed the logic of giving focus to the widget upon opening the context menu from `handleContextMenuEvent()` instead functionality added to handle it without focus
+  - This change prevents the context menu to trigger an activation for a widget
+  - While registering a command, `Event` should be passed down, if not passed, then it will break some context menu commands such as `Toggle Maximized`
```

Comment:
Isn't it more that if `Event` is not provided for tabbar context-menu items, the commands will not work correctly since they rely on the activation of the tabbar like in `master`. I'm not sure why the 'Toggle Maximized' command in particular was mentioned here.